### DISCUSSION TITLE
Windows Protected Me...

### DISCUSSION THREAD

**Discussion by BABingham** [martinet101/ElevenClock#821] [Q&A]
What would ELevenClock be named in the "Allow apps to communicate" so I can allow the update?

**Comment by marticliment** [comment]
Well, i don't know...

The domain name for the updates are `versions.somepythonthings.tk` and `www.somepythonthings.tk`

Can you post more details 'please?

**Comment by BABingham** [comment]
It just gave me a dialog box that said "Windows Protected You" , but after I updated windows it gave me the option for "More details" when this dialog box appeared.This was not there before. After I clicked that it allowed me to "Run anyway"I really do appreciate the quick response.If you need to know this appeared after I got the notification that there was a new update for ElevenClock.

    On Wednesday, July 6, 2022 at 07:13:59 AM CDT, Martí Climent ***@***.***> wrote:  
 
 


Well, i don't know...

The domain name for the updates are versions.somepythonthings.tk and www.somepythonthings.tk

Can you post more details 'please?

**Reply by marticliment** [reply]
If it works now, it is because it was shown as a false positive. This is a common thing to happen with unsigned apps, so much can't be done about it.

If it stuill does not work, please reply and we'll try to figure out the issue

**Comment by BABingham** [comment]
All good now, but thank you again!
    On Wednesday, July 6, 2022 at 01:54:27 PM CDT, Martí Climent ***@***.***> wrote:  
 
 


If it works now, it is because it was shown as a false positive. This is a common thing to happen with unsigned apps, so much can't be done about it.

If it stuill does not work, please reply and we'll try to figure out the issue
